hw/ip/tlul/generic_dv/xbar_sim_cfg.hjson

```diff
@@ -17,8 +17,8 @@
   // Fusesoc core file used for building the file list.
   fusesoc_core: "lowrisc:dv:{dut}_sim:0.1"
 
-  // Testplan hjson file: set in the autogenerated sim cfg file that imports this.
-  // testplan: ""
+  // Testplan hjson file.
+  testplan: "{proj_root}/hw/top_earlgrey/ip/{dut}/data/autogen/{dut}_testplan.hjson"
```

Comment:
I don't think its a good idea to put top level specific things in here. Any particular reason for moving these from the autogenerated sim cfg files?

Reply:
they're the same for all IPs. If we need to support other than top_earlgrey, we probably need to update or add a variable like {top}.

Reply:
Right, but it is easier to fix that in the template than here. Anyway, we can probably deal with it later.

Reply:
@sriyerg I reverted this back, this change can't support multiple top.